### PR TITLE
Add SensorDemo example for Nucleo-64 and X-NUCLEO-IDB0XA1

### PR DESCRIPTION
**Pull request by apalmieriGH**
@cparata @LMESTM @fpistm @avilei please review/merge this PR enabling Sensor Demo example for Nucleo-64 and X-NUCLEO-IDB0XA1

Thanks and regards
Andrea

### CODE REVIEW

File: examples/Nucleo64_SensorDemo/Nucleo64_SensorDemo.ino [hunk 1]
```diff
@@ -0,0 +1,146 @@
+/*
+
+ DISCO_IOT_SensorDemo
+
+ This sketch provides a default example how to use the BLE module of the
+ Discovery L475VG IoT board.
```
Comment (fpistm) [inline examples/Nucleo64_SensorDemo/Nucleo64_SensorDemo.ino]: Please update comment with shield name instead of board name

**Review by fpistm — COMMENTED** [review]
Rename the sketch with the shield name instead of Nucleo 64. It should be ok for Nucleo 144 and Disco also.

File: examples/BlueNRG_MS_SensorDemo/BlueNRG_MS_SensorDemo [hunk 2]
```diff
@@ -0,0 +1,146 @@
+/*
+
+ BlueNRG_MS_SensorDemo
```
Comment (VVESTM) [inline examples/BlueNRG_MS_SensorDemo/BlueNRG_MS_SensorDemo:3]: Need to add the .ino extension in the filename.

File: examples/BlueNRG_MS_SensorDemo/BlueNRG_MS_SensorDemo [hunk 3]
```diff
@@ -0,0 +1,146 @@
...
+{
+  /* Check if the user has pushed the button */
+  if(digitalRead(USER_BTN) == RESET)
+  {
+    while (digitalRead(USER_BTN) == RESET);
+
+    /* Update acceleration data */
+    p_axes->AXIS_X += 100;
+    p_axes->AXIS_Y += 100;
+    p_axes->AXIS_Z += 100;
```
Comment (VVESTM) [inline examples/BlueNRG_MS_SensorDemo/BlueNRG_MS_SensorDemo:143]: Same status as demo on L4 IOT Discovery board, the Z axis is not updated on the app side.

**Review by VVESTM — COMMENTED** [review]
Hi,
Few comments and need to add the .ino extension. Else,  test is OK for me.
As it is the same as the one done on Disco L4 IoT board, it can be good to align both. For example, keep only one file and add #define or comments to explain what changes to do to switch from one board to another. (changes are only pin naming).

Vincent

**Pull request merged**